cmd/snap-bootstrap/initramfs_systemd_mount_test.go

```diff
@@ -278,15 +331,21 @@ func (s *doSystemdMountSuite) TestDoSystemdMount(c *C) {
 					foundBeforeInitrdfsTarget = true
 				case strings.HasPrefix(arg, "--options="):
 					for _, opt := range strings.Split(strings.TrimPrefix(arg, "--options="), ",") {
-						switch opt {
-						case "nosuid":
+						switch opt := opt; {
```

Comment:
Is this assignment in the switch needed?

Reply:
It's not. Thanks, removed.